### PR TITLE
Fix getScale()

### PR DESCRIPTION
**Pull request by jordan4ibanez**

### CODE REVIEW

File: source/Irrlicht/CGLTFMeshFileLoader.cpp [hunk 1]
```diff
@@ -311,13 +311,16 @@ void CGLTFMeshFileLoader::MeshExtractor::copyTCoords(
  * Required: NO
  * @returns: core::vector2df
 */
-float CGLTFMeshFileLoader::MeshExtractor::getScale() const
+core::vector3df CGLTFMeshFileLoader::MeshExtractor::getScale() const
 {
-	if (m_model.nodes[0].scale.size() > 0) {
-		return static_cast<float>(m_model.nodes[0].scale[0]);
+	core::vector3df buffer{1,1,1};
```
Comment (JosiahWI) [inline source/Irrlicht/CGLTFMeshFileLoader.cpp]:
```suggestion
	core::vector3df buffer{1.0f,1.0f,1.0f};

```

File: source/Irrlicht/CGLTFMeshFileLoader.cpp [hunk 2]
```diff
@@ -311,13 +311,16 @@ void CGLTFMeshFileLoader::MeshExtractor::copyTCoords(
  * Required: NO
  * @returns: core::vector2df
 */
-float CGLTFMeshFileLoader::MeshExtractor::getScale() const
+core::vector3df CGLTFMeshFileLoader::MeshExtractor::getScale() const
 {
-	if (m_model.nodes[0].scale.size() > 0) {
-		return static_cast<float>(m_model.nodes[0].scale[0]);
+	core::vector3df buffer{1,1,1};
+	//! Fixme: What if there is more than one node?
```
Comment (JosiahWI) [inline source/Irrlicht/CGLTFMeshFileLoader.cpp]: Please remove the comment to avoid confusion, since we do not plan to support more than one node.

**Review by JosiahWI — CHANGES_REQUESTED** [review]
Thank you for adding the unit test! I have a few small nitpicks.

File: source/Irrlicht/CGLTFMeshFileLoader.cpp (hunk 1 above)
Comment (JosiahWI) [reply source/Irrlicht/CGLTFMeshFileLoader.cpp]: This does not look resolved to me.

File: source/Irrlicht/CGLTFMeshFileLoader.cpp (hunk 2 above)
Comment (jordan4ibanez) [reply source/Irrlicht/CGLTFMeshFileLoader.cpp]: Fixed in 036c72cc5424d7868dfdcff08ccc6a830c7de1a9

**Review by JosiahWI — APPROVED** [review]

File: source/Irrlicht/CGLTFMeshFileLoader.cpp (hunk 1 above)
Comment (jordan4ibanez) [reply source/Irrlicht/CGLTFMeshFileLoader.cpp]: Fixed in a95861a8487cc847e65a7beffdf5e7c1cc2d95d4

**Pull request merged**